api/internal/git/repospec_test.go

```diff
@@ -191,24 +195,37 @@ func TestNewRepoSpecFromUrl_Smoke(t *testing.T) {
 		},
 		{
 			name:      "t6",
-			input:     "git@gitlab2.sqtools.ru:10022/infra/kubernetes/thanos-base.git?ref=v0.1.0",
```

Comment:
This test was invalid. `git fetch` would've treated `10022` as a folder name, not a port.